### DISCUSSION TITLE
Scrollbar doesn't show on mobile devices

### DISCUSSION THREAD

**Discussion by meness** [tailwindlabs/tailwindcss#11422] [Help]
**What version of Tailwind CSS are you using?**
3.3.2

**What build tool (or framework if it abstracts the build tool) are you using?**
Not sure

**What version of Node.js are you using?**
Not sure

**What browser are you using?**
Chrome on iOS, Chrome on Android

**What operating system are you using?**
iOS and Android

**Reproduction URL**
https://play.tailwindcss.com/8s7SKiOFr8

**Describe your issue**
The scrollbar shows as expected on desktop browsers, but it doesn't show on mobile devices. As you can see in the reproduction URL above, I wrote a very simple HTML. I expect that works everywhere

**Comment by thecrypticace** [comment]
This is because mobile devices only show scroll bars when scrolling is actively happening. On iOS, as far as I know, there's no option to always show them. I'm uncertain about Android though.

**Reply by meness** [reply]
When I posted the issue here, I couldn't see the scrollbar while scrolling on iOS or Android. Now, the scrollbar shows on iOS when scrolling, but it still not showing on Android

**Reply by mybearworld** [reply]
Mobile devices often don't have scroll bars.

**Comment by KESIE07** [comment]
Thank you for your help! [myfedloan](https://www.myfedloan.us/)
